docs/release/release-templates.md

```diff
@@ -0,0 +1,63 @@
+# Release Templates
+
+This document contains a collection of announcement templates for the release team.
+
+## Email Release Announcement Template
+
+```
+Hello everyone,
+
+Cluster API patch version vX.Y.Z has been released!
+
+This release was focused on improving existing features and stability of CAPI. 
+There were a total of A commits and B bugs fixed by our awesome contributors! Kudos!
+
+Below is the release note of the vX.Y.Z patch release:
+- https://github.com/kubernetes-sigs/cluster-api/releases/tag/vX.Y.Z
+
+Thanks to all our contributors!
+
+Best, 
+CAPI Release Team
+```
+
+## Slack Weekly Announcement Template
+
+```
+# Weekly announcement template
```

Comment:
Yes although would it still be worth keeping around if someone wants to reference it? I don't see why anyone would need to though so I can go ahead and remove it.

Reply:
I'd probably leave it as a reference, since someone new in the team to the tool and never used it before to generate weekly announcements that will be introduced in #9343, has an idea of how it looks like beforehand.

Reply:
I just think it's going to be a lot of effort to keep it in sync vs just running the tool. Maybe the compromise is to have the templates for the notes tool be templates somewhere instead of being inlined in the go code?

Reply:
I can add a line specifying that you can generate this template automatically with the notes tool. It might take some significant refactoring to have the generator to use it. Should I work on it now or defer it to a separate issue?

Reply:
Fine to keep it for now IMO if the team is happy with it - would be good to move to templating over time as it probably makes the whole process easier to change and adapt.